pootle/static/js/editor/components/SuggestionFeedbackForm.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import FormElement from 'components/FormElement';
+import FormMixin from 'mixins/FormMixin';
+
+import { decodeEntities } from '../utils';
+
+export const SuggestionFeedBackForm = React.createClass({
+
+  propTypes: {
+    suggId: React.PropTypes.number.isRequired,
+    suggText: React.PropTypes.string.isRequired,
+    editor: React.PropTypes.object.isRequired,
+  },
+
+  mixins: [FormMixin],
+
+  /* Lifecycle */
+
+  getInitialState() {
+    return {
+      formData: {
+        comment: '',
+        translation: this.props.suggText,
```

Comment:
As suggested by the official docs, generally speaking [props in `getInitialState()` is an anti-pattern](https://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html). This one is a valid use, however `suggText` should be more explicit in its intent, e.g renaming it to `initialSuggestionText`.